src/components/FieldLevelHelp/FieldLevelHelp.js

```diff
@@ -1,63 +1,51 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Icon } from '../Icon';
+import { Button } from '../Button';
 import { Popover } from '../Popover';
-import { Tooltip } from '../Tooltip';
 import { OverlayTrigger } from '../OverlayTrigger';
 
 /**
  * FieldLevelHelp Component for Patternfly React
  */
-const FieldLevelHelp = ({ children, mode, content, close, ...props }) => {
-  const trigger = mode === 'popover' ? 'click' : 'hover focus';
+const FieldLevelHelp = ({ children, content, close, ...props }) => {
+  const trigger = 'click';
   const htmlContent = (
     <div
       dangerouslySetInnerHTML={{
         __html: content
       }}
     />
   );
-  const overlay =
-    mode === 'popover' ? (
-      <Popover id="{mode}">{htmlContent}</Popover>
-    ) : (
-      <Tooltip id="{mode}">{htmlContent}</Tooltip>
-    );
+  const overlay = <Popover id="popover">{htmlContent}</Popover>;
   const rootClose = close === 'true';
 
   return (
-    <div>
-      <label>{`${children} `}</label>
-      <OverlayTrigger
-        overlay={overlay}
-        placement={'top'}
-        trigger={trigger.split(' ')}
-        rootClose={rootClose}
+    <OverlayTrigger
+      overlay={overlay}
+      placement={'top'}
+      trigger={trigger.split(' ')}
+      rootClose={rootClose}
+    >
+      <Button
+        bsStyle="link"
+        style={{ textDecoration: 'none', outline: 'none' }}
```

Comment:
Can we remove these styles? Removing the outline results in an accessibility issue, because focus is not obvious to the sighted user who uses a keyboard to navigate. 

Also, as a general rule, if styles are ever needed, chances are they should be added to the core patternfly repo so that all JS repos can benefit from them. There are probably rare cases where react has a unique case that exists only in the context of the react repo. In that case we would add .less and .scss files for the component. I just now opened PR #195 to the [contributing guide](https://github.com/jgiardino/patternfly-react/blob/contributing/CONTRIBUTING.md#adding-styling-for-your-components) that includes some of these updates, in case you want to review and provide feedback :-)